homeassistant/components/homekit_controller/sensor.py

```diff
@@ -203,17 +205,17 @@ class HomeKitHumiditySensor(HomeKitEntity, SensorEntity):
     _attr_device_class = SensorDeviceClass.HUMIDITY
     _attr_native_unit_of_measurement = PERCENTAGE
 
-    def get_characteristic_types(self):
+    def get_characteristic_types(self) -> list[str]:
         """Define the homekit characteristics the entity is tracking."""
         return [CharacteristicsTypes.RELATIVE_HUMIDITY_CURRENT]
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the device."""
         return f"{super().name} Humidity"
 
     @property
-    def native_value(self):
+    def native_value(self) -> StateType:
```

Comment:
Can this actually be `Union[None, str, int, float]` ?

Reply:
Sorry if its a dumb question, but isn't StateType exactly that?

```
StateType = Union[None, str, int, float]
```

Is it better to not use an alias for some reason?

Reply:
https://github.com/home-assistant/core/pull/65368#discussion_r796792064